### PR TITLE
deps: updates main versions notably zipkin

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This changes scribe tests to not share a dependency with zipkin, but it doesn't address #201, which breaks compatibility.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
sorry scribe got upset

**Comment by codefromthecrypt** [comment]
I think I'll switch the scribe test to use docker so the classpath isn't tangled with zipkin's

File: libthrift/src/test/java/zipkin2/reporter/libthrift/ITLibthriftSender.java [hunk 1]
```diff
@@ -17,100 +17,93 @@
...
+  @Test void sendsSpans_multipleTimes() throws Exception {
     for (int i = 0; i < 5; i++) { // Have client send 5 messages
       send(Arrays.copyOfRange(LOTS_OF_SPANS, i, (i * 10) + 10));
     }
 
-    assertThat(storage.getTraces()).flatExtracting(l -> l)
-      .contains(Arrays.copyOfRange(LOTS_OF_SPANS, 0, 50));
-  }
-
-  @Test public void sendsSpansExpectedMetrics() throws Exception {
```
Comment (codefromthecrypt) [inline libthrift/src/test/java/zipkin2/reporter/libthrift/ITLibthriftSender.java:76]: I removed this as it was testing zipkin not zipkin-reporter.

File: pom.xml [hunk 2]
```diff
@@ -67,7 +67,8 @@
     <junit.version>4.13.2</junit.version>
     <junit-jupiter.version>5.10.1</junit-jupiter.version>
     <okhttp.version>3.14.9</okhttp.version>
-    <logback.version>1.4.14</logback.version>
+    <!-- use pre slf4j 2 version -->
+    <logback.version>1.2.13</logback.version>
```
Comment (codefromthecrypt) [inline pom.xml:71]: reverted noticing logging didn't work

**Review by anuraaga — APPROVED** [review]

**Pull request merged**